lore/encoders.py

```diff
@@ -575,18 +575,31 @@ def cardinality(self):
 
 class OneHot(Base):
     """
-    Performs one hot encoding
+    Performs one hot encoding. One hot encoding is used to encode categorical variables into a series of indicator variables.
     """
-    def __init__(self, column, name=None, minimum_occurrences=None, compressed=False, **kwargs):
-        if compressed is True and minimum_occurrences is None:
-            raise ValueError('minimum_occurrences must be specified when compressed is True')
-        elif compressed is False and minimum_occurrences is not None:
-            logger.warning('minimum_occurrences has no effect when compressed is False')
+    def __init__(self, column, name=None, minimum_occurrences=None, percent_occurrences=None,
```

Comment:
If I understand this correctly what we are doing is if the categorical variable count or % wise is less than some factor x we just set them to 0? Can you point me to some more information about the intuition behind this?

Reply:
In one hot encoding, each of the variables represents the additional effect of that categorical level. If you have "rare" levels, you are essentially saying that they are not contributing any "additional" effect. That's one way to look at it. Alternatively, you could say that the rare levels are being pooled into one common level.